core/types/src/attributes.rs

```diff
@@ -54,7 +54,7 @@ impl Display for Attributes {
             }
             None => write!(f, "Flags: (none)")?,
         }
-        match Xfrm::from_bits(self.0.xfrm) {
+        match ExtendedFeatureRequestMask::from_bits(self.0.xfrm) {
```

Comment:
I'm fine if people think this should go back to `Xfrm` to match Intel's usage.
My background has me see `Xfrm` as "transform" and doing a quick google search for xfrm IP transforms seem to come up the most.

Reply:
I like the explicit name, being new to looking at a lot of SGX terminology and all the obscure acronyms involved.